src/pages/site/[uuid]/tabs/Overview.tsx

```diff
@@ -110,6 +111,13 @@ const SiteOverviewTab = ({ site, refetch: refetchEntity }: SiteOverviewTabProps)
     }
   }, [files, saveFlags]);
 
+  useEffect(() => {
+    if (errorMessage) {
+      openNotification("error", t("Error uploading file"), t(errorMessage));
+      setErrorMessage(null);
+    }
+  }, [errorMessage]);
+
```

Comment:
I don't see a need for this effect or errorMessage state variable. Can't the `setErrorMessage` callback below just call open notification directly? 
```
setErrorMessage={errorMessage => openNotification("error", t("Error uploading file"), t(errorMessage)}
```

Reply:
Done. the const and the UseEffect have been removed.